### ISSUE TITLE
how can I update to beta V.0.5.11??

### ISSUE THREAD

**Issue by ellimon** [loqui/im#1182]
hi everybody! I would want know-how update my loqui im to the latest version beta V.0.5.11 ¿¿ somebody can you help me?? pleaseee

**Comment by nfsprodriver** [comment]
What platform do you use? Firefox OS or Ubuntu Touch?

**Comment by ellimon** [comment]
firefox OS

**Comment by nfsprodriver** [comment]
I don't want to say something wrong. I'm UT user. But as far I know you may download the firefoxos zip file from the releases and install it to your device via the browser's WebIDE. Mozilla gives more instructions for this.

**Comment by ellimon** [comment]
thanks!

**Comment by Zeus76** [comment]
You watch here: https://www.facebook.com/415394908611505/photos/pcb.684050378412622/684050081745985/?type=3&theater

**Comment by ibinshoid** [comment]
hello,
i have installed loqui 0.5.12 on firefoxOS 1.3 where 0.5.10 was installed. But now i have both versions on my phone and all chats are in loqui 0.5.10. is there a way to update from one version to another or copy the data to loqui 0.5.12?

**Comment by Honk2** [comment]
You need to connect your phone to a PC, Start Firefox + WebIde.
Connect to your device.
Select Loqui on the left side and open the console (press the wrench icon)
In the Console type App.exportData() do not omit the braces!
uninstall 5.10
install 5.11
Select Loqui in webide / start Loqui
open Console and enter App.importData()
Thats it

You can also imstall the FFOS emulator Addon and test the preocedure before doing it ob your phone

**Comment by urrfaust** [comment]
`App.exportData()` 
returns
"undefined"
does it work at all?

**Comment by Honk2** [comment]
Yes it does work. Look at your phone after typing into the console.
You are requested to enter a password to encrypt the export.
The export starts after you typed in the password.
Be carefull to remember the password, you will need it again for import.
And Loqui does not ask to type the password two time to verify it.

**Comment by ibinshoid** [comment]
Thank you very much.
This is also a good way for backup my data.

**Comment by Honk2** [comment]
That is the intended use of exportData() ;)

**Comment by urrfaust** [comment]
Yep that worked. Thanks a lot :-)

**Comment by madtibo** [comment]
Version 0.5.17 installed and running on FFOS 2.5. Thank you very much!

(just for information, the version 0.5.23 is not working on FFOS 2.5).
